types/enzyme-adapter-react-16/index.d.ts

```diff
@@ -9,7 +10,5 @@ import { EnzymeAdapter } from 'enzyme';
 declare class ReactSixteenAdapter extends EnzymeAdapter {
 }
 
-declare namespace ReactSixteenAdapter {
-}
-
-export = ReactSixteenAdapter;
+declare const Adapter: typeof ReactSixteenAdapter;
```

Comment:
You also don't need this const declaration? `export = ReactSixteenAdapter;` should result in same?

Reply:
Hi @sheetalkamat, interestingly that's the only functional change I was trying to make, as without it you get the compilation error:

`Module '"c:/Git/DefinitelyTyped/types/enzyme-adapter-react-16/index"' resolves to a non-module entity and cannot be imported using this construct.`

I've tried various techniques for declaring the export, importing and configuring TSC, but this approach is the only one I could get to work.  Unless you've any suggestions?

Reply:
I don't know why that would be error. Can you please make the change and if travis build fails we can take a look.